sdk/template-LLC/Azure.Template.LLC/src/swagger/swagger.json

```
@@ -1,123 +1,121 @@
 ﻿{
   "swagger": "2.0",
   "info": {
-    "title": "Template LLC",
-    "description": "Template LLC",
+    "title": "Template Service",
+    "description": "Template Service",
     "version": "1.0.0"
   },
   "host": "localhost:3000",
-  "schemes": ["http"],
-  "produces": ["application/json"],
-  "consumes": ["application/json"],
+  "schemes": [
+    "http"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "consumes": [
+    "application/json"
+  ],
   "paths": {
-    "/test1": {
-      "get": {
-        "operationId": "NoRequestBodyResponseBody",
-        "description": "No RequestBody and ResponseBody",
+    "/providers/Microsoft.Template/resource/{resourceId}": {
```

```suggestion
    "/template/resources/{resourceId}": {
```
I think it would be better not to use management plane path formats, since this is a template for data-plane llc.  We should also use plural for the collection name (i.e. resources instead of resource), since this is the REST API convention -- see [Azure REST API guidelines on Collections](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md#collections)